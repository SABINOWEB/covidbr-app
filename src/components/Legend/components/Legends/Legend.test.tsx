import React from 'react'
import { Legend } from './index'
import { render } from '@testing-library/react'

describe('<Legend />', () => {
  test('deve renderizar corretamente uma legenda do tipo "line"', () => {
    const { container } = render(
      <Legend
        indicatorType='line'
        label='Casos'
        total={10}
        color='#000000'
      />
    )

    expect(container).toMatchSnapshot()
  })

  test('deve renderizar corretamente uma legenda do tipo "circle"', () => {
    const { container } = render(
      <Legend
        indicatorType='circle'
        label='Casos'
        total={10}
        color='#000000'
      />
    )

    expect(container).toMatchSnapshot()
  })
})
