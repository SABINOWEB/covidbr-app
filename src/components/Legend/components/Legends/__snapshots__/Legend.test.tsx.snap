// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Legend /> deve renderizar corretamente uma legenda do tipo "circle" 1`] = `
<div>
  <div
    class="sc-AxhUy gWaiz"
  >
    <div
      class="sc-AxheI jCCqEc"
      color="#000000"
    />
    <div
      class="sc-AxirZ fvzfAf"
    >
      <span
        class="sc-AxiKw boVvlH"
      >
        Casos
      </span>
      <span
        class="sc-AxjAm sc-AxhCb JcUDR"
      >
        10
      </span>
    </div>
  </div>
</div>
`;

exports[`<Legend /> deve renderizar corretamente uma legenda do tipo "line" 1`] = `
<div>
  <div
    class="sc-AxhUy gWaiz"
  >
    <div
      class="sc-AxgMl gtSLvA"
      color="#000000"
    />
    <div
      class="sc-AxirZ fvzfAf"
    >
      <span
        class="sc-AxiKw boVvlH"
      >
        Casos
      </span>
      <span
        class="sc-AxjAm sc-AxhCb JcUDR"
      >
        10
      </span>
    </div>
  </div>
</div>
`;
